bitnami/postgresql-repmgr/12/debian-12/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -485,11 +485,7 @@ repmgr_postgresql_configuration() {
     else
         repmgr_inject_postgresql_configuration
     fi
-    if repmgr_is_file_external "pg_hba.conf"; then
-        cp "${REPMGR_MOUNTED_CONF_DIR}/pg_hba.conf" "${POSTGRESQL_MOUNTED_CONF_DIR}/pg_hba.conf"
-    else
-        repmgr_inject_pghba_configuration
-    fi
+    repmgr_inject_pghba_configuration
```

Comment:
A comment explaining why the default pghba file is injected even when custom one is provided may be useful.
Could you please apply this suggestion in all the files?
```suggestion
    # Inject default pghba configuration
    # May be override by custom ph_hba.conf later after initialization
    repmgr_inject_pghba_configuration
```